### ISSUE TITLE
Figure out a way to use custom (imported) fonts with mailing-ui

### ISSUE THREAD

**Issue by ericvalcik** [webscopeio/mailingui-web#131]
Last time I checked this, I tried to use the `react.email` Font component, but didn't find much luck. Will try to investigate this again now

**Comment by iamhectorsosa** [comment]
So just to understand, if I want to use Roboto from Google fonts. This component didn't work?

https://react.email/docs/components/font

**Comment by ericvalcik** [comment]
I had trouble setting it up but I didn't look into it that much. I believe it deserves a second look, so I will investigate

**Comment by ericvalcik** [comment]
Support for webfonts (`@font-face`), that could allow us to host fonts somewhere and use them in the email is very low:

![image](https://github.com/webscopeio/mailingui-web/assets/48859243/696240fb-d590-4284-8cb2-5ca6cc71b7a6)

And another thing I found is that using the `Font` component from `@react-email/components` does not work. For some reason, in the rendered HTML, a lot of `&#x27` hex chars is present. The code below is the generated HTML when importing `Font` from `@react-email/component`.

```html
<head data-id="__react-email-head">
  <style>
    @font-face {
      font-family: &#x27;
      Ubuntu&#x27;
      ;
      font-style: normal;
      font-weight: 400;
      mso-font-alt: &#x27;
      Verdana&#x27;
      ;
      src: url(http://fonts.gstatic.com/s/ubuntu/v7/c2KTkZ_oLp2UOMwi4GGs9A.woff) format(&#x27; woff&#x27; );
    }
    * {
      font-family: &#x27;
      Ubuntu&#x27;
      ,
      Verdana;
    }
  </style>
</head>
```

However, I when I look the source code of `react-email` and copied the `Font` component into our project from the `main` branch ([link to the code](https://github.com/resendlabs/react-email/blob/main/packages/font/src/font.tsx)), and used the copy and pasted component instead of the one in the `@react-email/components` package, suddenly it worked.

```html
<head data-id="__react-email-head">
  <style>
    @font-face {
      font-family: 'Ubuntu';
      font-style: normal;
      font-weight: 400;
      mso-font-alt: 'Verdana';
      src: url(http://fonts.gstatic.com/s/ubuntu/v7/c2KTkZ_oLp2UOMwi4GGs9A.woff) format('woff');
    }
    * {
      font-family: 'Ubuntu', Verdana;
    }
  </style>
</head>
```

No more `&#x27` hex chars and also the paragraph renders correctly.

![image](https://github.com/webscopeio/mailingui-web/assets/48859243/c53d8c04-f092-482e-8ec2-161b8d7cfcdc)

---

In conclusion, the `@react-email` Font renders wrongly, but even if we managed to get it working somehow, we would not gain that much. The only benefit would be the webfonts, but they are not supported mostly so I believe we should avoid them anyway, and just using `fontFamily` tag in our templates serves the same purpose than using their component.

IMO, I'm fine with just sticking to plain old `font-family` inline tag or maybe creating our own component.

Wdyt @iamhectorsosa ?

**Comment by ericvalcik** [comment]
We can discuss this more, but I think we can't really do much with webfonts, and just using `fontFamily` is our best bet right now.

I will close this issue, but feel free to reopen this issue if you want to discuss it.
